File: src/main/java/dev/jsinco/brewery/MyConfig.java
package dev.jsinco.brewery;

import com.dre.brewery.api.addons.AddonConfigFile;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class MyConfig extends AddonConfigFile {

    // OR You can use com.dre.brewery.depend.okaeri.configs.annotation.Exclude annotation
    private transient boolean someFieldWhichShouldBeIgnored = true;


    private String myString = "Hello, World!";
}

File: src/main/java/dev/jsinco/brewery/MyConfig.kt
package dev.jsinco.brewery

import com.dre.brewery.api.addons.AddonConfigFile


class MyConfig : AddonConfigFile() {
    // OR You can use com.dre.brewery.depend.okaeri.configs.annotation.Exclude annotation
    @Transient
    private val someFieldWhichShouldBeIgnored = true


    @JvmField // Instructs the Kotlin compiler not to generate getters/setters for this property and expose it as a field.
    val myString = "Hello, World!"
}
